k9mail-library/src/main/java/com/fsck/k9/mail/oauth/OAuth2TokenProvider.java

```diff
@@ -0,0 +1,61 @@
+package com.fsck.k9.mail.oauth;
+
+import android.app.Activity;
+
+import com.fsck.k9.mail.AuthenticationFailedException;
+
+import java.util.List;
+
+public interface OAuth2TokenProvider {
```

Comment:
It's not. This is just the back-end changes. That's an interface for the front-end to implement.